src/Symfony/Component/Validator/Tests/Constraints/EmailValidatorTest.php

```diff
@@ -94,23 +93,23 @@ public function getInvalidEmails()
         );
     }
 
-    public function testStrict()
+    public function testRfc()
```

Comment:
same: let's keep strict everywhere to reduce the diff, thus help the review+future merges